src/Commands/Generator.php

```diff
@@ -152,6 +160,7 @@ protected function generate(string $type, string $parentClass = ''): bool
         $namespace = $this->pathResolver->typeNamespace($type);
         $lcModelName = Str::lower($this->modelName);
         $lcPluralModelName = Str::plural($lcModelName);
+        $ucPluralModelName = Str::plural(Str::ucfirst($this->modelName));
```

Comment:
I think we will not need this variable if we pass namespaces to stub (check next comments)